revocation/ocsp/errors.go

```diff
@@ -0,0 +1,58 @@
+// Package OCSP provides methods for checking the OCSP revocation status of a certificate chain
+// as well as errors related to these checks
+package ocsp
```

Comment:
If ocsp will return the errors from revocation, then we would have a cyclic dependency (since ocsp would rely on revocation for errors, and revocation would rely on ocsp for OCSPStatus)